cmd/chart.go

```diff
@@ -100,21 +97,9 @@ func loadImagePatterns(chart *chart.Chart) (string, error) {
 }
 
 func loadRules() (*rewrite.Rules, error) {
-	rules := &rewrite.Rules{}
-	if RulesFile != "" {
-		var err error
-		rules, err = rewrite.ParseRules(RulesFile)
-		if err != nil {
-			return nil, err
-		}
-	}
-
-	if RegistryRule != "" {
-		rules.Registry = RegistryRule
-	}
-
-	if RepositoryPrefixRule != "" {
-		rules.RepositoryPrefix = RepositoryPrefixRule
+	rules := &rewrite.Rules{
+		Registry:         RegistryRule,
+		RepositoryPrefix: RepositoryPrefixRule,
 	}
 
 	return rules, nil
```

Comment:
I didn't remove it because it's part of my other patch. The same applies to not to remove the rewrites package. Note how if I do that I'll eventually end up in the same shape than the other patch haha

Reply:
What I mean is that I was planning on adding another patch that includes the removal of the rewrites package, the refactoring around the Rules/OCILocation structs which will affect the code you just described.

In any case, to your specific piece of comment, I can do it, take a look

Reply:
Ok, good enough